src/lib/doctrine-dbal-bulk/tests/Flow/Doctrine/Bulk/Tests/Unit/BulkDataTest.php

```diff
@@ -99,11 +100,11 @@ public function test_returns_all_sql_parameters_as_one_dimensional_array_with_pl
             $bulkData->toSqlParameters(
                 new TableDefinition(
                     'test',
-                    new Column('date', new StringType()),
-                    new Column('title', new StringType()),
-                    new Column('description', new StringType()),
-                    new Column('quantity', new IntegerType()),
-                    new Column('errors', new JsonType()),
+                    new Column('date', $registry->get(Types::STRING)),
```

Comment:
This is required due to the fact new registry holds the hash of type objects, and creating it here returns a different hash which ends with an exception due to mismatch.